### PR TITLE
Remove codecov and use coveralls

### LINKED ISSUE #72: Fix broken code cov
CodeCov has not been updated in months:
https://codecov.io/gh/vanniktech/gradle-android-junit-jacoco-plugin/commits

See the latest CI build:
https://travis-ci.org/vanniktech/gradle-android-junit-jacoco-plugin#L364

```
    Executing gcov (find /home/travis/build/[secure]/gradle-android-junit-jacoco-plugin -type f -name '*.gcno'  -exec gcov -pb  {} +)
==> Collecting reports
Error: No coverage report found
Tip: See an example jdk repo: https://github.com/codecov/example-jdk
Support channels:
  Email:   hello@codecov.io
  IRC:     #codecov
  Gitter:  https://gitter.im/codecov/support
  Twitter: @codecov
Done. Your build exited with 0.
```

### PR DESCRIPTION
**Pull request by jaredsburrows**
@vanniktech 
Closes #72

### CODE REVIEW

**Comment by jaredsburrows** [comment]
It does not seem to generate it?
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><!DOCTYPE report PUBLIC "-//JACOCO//DTD Report 1.0//EN" "report.dtd"><report name="gradle-android-junit-jacoco-plugin"><sessioninfo id="nonames-MacBook-Pro.local-98f65650" start="1501444222620" dump="1501444232015"/></report>
```

**Comment by jaredsburrows** [comment]
@vanniktech Using the Jacoco plugin directly generates the code coverage.

**Comment by jaredsburrows** [comment]
@vanniktech It looks like your current plugin does not support Groovy: https://github.com/vanniktech/gradle-android-junit-jacoco-plugin/blob/master/src/main/groovy/com/vanniktech/android/junit/jacoco/GenerationPlugin.groovy#L71

**Comment by vanniktech** [comment]
Whops might be time to change that. Same for `src/main/kotlin`.

File: build.gradle [hunk 1]
```diff
@@ -77,3 +77,4 @@ artifacts {
 }
 
 apply from: file('gradle/gradle-mvn-push.gradle')
+apply from: file('gradle/quality.gradle')
```
Comment (vanniktech) [inline build.gradle]: let's go and move everything inside here. just restore the top line:

```diff
-apply plugin: 'com.vanniktech.android.junit.jacoco'
```

and add the content of the `quality.gradle` file

**Comment by jaredsburrows** [comment]
@vanniktech Should be good to go now.

**Comment by jaredsburrows** [comment]
@vanniktech Ok to merge?

**Comment by vanniktech** [comment]
Doesn't this need the other PR first? Also I'm tempted to maybe stay with codecov if the reason why there were no reports is the issue pointed out in #75, mainly because then I'd have to migrate a lot of projects from codecov to coveralls.

**Comment by jaredsburrows** [comment]
@vanniktech Ok, it's up to you.

**Comment by jaredsburrows** [comment]
@vanniktech Should I close this?

**Comment by vanniktech** [comment]
I'll release a new version tonight and then we'll see whether this gets resolved or not.

**Comment by jaredsburrows** [comment]
@vanniktech Sounds good.

**Comment by jaredsburrows** [comment]
Closing this and #72 because https://github.com/vanniktech/gradle-android-junit-jacoco-plugin/pull/78 fixed the problem.

**Comment by jaredsburrows** [comment]
@vanniktech Merge?

**Comment by jaredsburrows** [comment]
I am not sure. Merge this in and configure it to do so. Why are you resisting this PR so much?

**Comment by jaredsburrows** [comment]
Alright. The badge should update once there is coverage for the `master` branch.

**Pull request merged**